packages/driver/src/cy/commands/aliasing.ts

```diff
@@ -22,9 +20,10 @@ export default function (Commands, Cypress, cy, state) {
         })
       }
 
+      // TODO: Change any to Log type.
```

Comment:
Might be good to have a more descriptive TODO here in case someone else comes across this without context of the ongoing changes.
```suggestion
      // TODO: change the log type from `any` to `Log`.
```

Reply:
Done.